crates/nu-cli/src/completion/path.rs

```diff
@@ -73,7 +74,12 @@ impl PathCompleter {
 }
 
 impl Completer for PathCompleter {
-    fn complete(&self, _ctx: &Context<'_>, partial: &str) -> Vec<Suggestion> {
+    fn complete(
+        &self,
+        _ctx: &Context<'_>,
+        partial: &str,
+        _matcher: &dyn Matcher,
```

Comment:
You'll want to make this part of `path_suggestions` too (note the `file_name.starts_with(partial)` it has).

Reply:
Ah, thanks. fixing now.